jekyll/_cci2/configuration-reference.md

```diff
@@ -695,7 +695,7 @@ Configured by `docker` key which takes a list of maps:
 Key | Required | Type | Description
 ----|-----------|------|------------
 image | Y | String | The name of a custom docker image to use. The first `image` listed under a job defines the job's own primary container image where all steps will run.
-name | N | String | `name` defines the name for reaching the secondary service containers.  By default, all services are exposed directly on `localhost`.  The field is appropriate if you would rather have a different host name instead of localhost, for example, if you are starting multiple versions of the same service.
+name | N | String | `name` defines the the hostname for the container (default: localhost), used for reaching secondary, service containers. By default, all services are exposed directly on `localhost`. This field is appropriate if you would rather have a different hostname instead of `localhost`, for example, if you are starting multiple versions of the same service.
```

Comment:
```suggestion
name | N | String | `name` defines the the hostname for the container (the default is `localhost`), used for reaching secondary, service containers. By default, all services are exposed directly on `localhost`. This field is appropriate if you would rather have a different hostname instead of `localhost`, for example, if you are starting multiple versions of the same service.
```